java/grpc-client/src/test/java/com/youtube/vitess/client/grpc/GrpcClientTest.java

```diff
@@ -41,7 +42,9 @@ public static void setUpBeforeClass() throws Exception {
                     Integer.toString(port),
                     "-service_map",
                     "grpc-vtgateservice"))
+            .inheritIO()
             .start();
+    Thread.sleep(TimeUnit.SECONDS.toMillis(10));
```

Comment:
Do we still need this sleep and the one below? It seems the java grpc client will retry until timeout if the server is not there yet, so we don't need to wait?